ui/src/app/settings/containers/Configuration/Configuration.js

```diff
@@ -1,5 +1,29 @@
 import React from "react";
+import { useSelector } from "react-redux";
 
-const Configuration = () => <h4>Configuration</h4>;
+import selectors from "app/settings/selectors";
+import Loader from "app/base/components/Loader";
+import GeneralForm from "app/settings/components/GeneralForm";
+
+const Configuration = () => {
+  const loaded = useSelector(selectors.config.loaded);
+  const loading = useSelector(selectors.config.loading);
```

Comment:
Hi @steverydz, please don't pass state down into children, instead, call selectors where the state is used (in `GeneralForm`).

Have a look at [Should I only connect my top component, or can I connect multiple components in my tree?](https://redux.js.org/faq/react-redux#should-i-only-connect-my-top-component-or-can-i-connect-multiple-components-in-my-tree)